### PR TITLE
Correct code in before pr and fix an error in about page

### PR DESCRIPTION
**Pull request by HanielF**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [x] The commit message follows [our guidelines](https://github.com/iissnan/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [ ] Tests for the changes have been added (for bug fixes / features).
- [ ] Docs have been added / updated (for bug fixes / features).

## PR Type
**What kind of change does this PR introduce?**  

- [x] Bugfix.
- [ ] Feature.
- [x] Code style update (formatting, local variables).
- [ ] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [ ] Documentation content changes.
- [ ] Other... Please describe:

## What is the current behavior?

The component of footer is floating without fixed.

Issue Number(s): N/A

## What is the new behavior?
Description about this pull, in several words...

Keep the footer at the bottom in about page.    

Correct code in previous pr.

* Screens with this changes: N/A

* Link to demo site with this changes: N/A

## Does this PR introduce a breaking change?
- [ ] Yes.
- [x] No.

### CODE REVIEW

**Comment by HanielF** [comment]
I'm still not sleepy... but I have to...

File: _config.yml [hunk 1]
```diff
@@ -91,8 +91,12 @@ scheme: Mala
 # Social Links
 # Key is the link label showing to end users.
 # Value is the target link (E.g. GitHub: https://github.com/iissnan)
-#social:
+social:
+
   #LinkLabel: Link
+  Github: https://github.com/HanielF
```
Comment (Gabirel) [inline _config.yml]: Personal data included in this PR. Why?

Comment (HanielF) [reply _config.yml]: Ummm,u really want to know? 
because I remembered it until i pulled request...

**Comment by HanielF** [comment]
It should be alright now.

**Pull request merged**